Monika After Story/game/script-story-events.rpy

```diff
@@ -8,11 +8,12 @@ init 5 python:
 
 label gender:
     #TODO: update exp's on this
-    m 2d "...[player]? So I've been thinking a bit."
-    m "I've mentioned before that the 'you' in the game might not reflect the real you."
-    m 1m "But I guess I had just assumed that you were probably a guy."
-    m "The main character was, after all."
-    m 1a "But if I'm going to be your girlfriend, I should probably know at least this much about the real you."
+    # done
+    m 2eud "...[player]? So I've been thinking a bit."
```

Comment:
Too many `d` mouths, and there's already a branch open for fixing this stuff, which we'll do there.